gcp_variant_transforms/testing/integration/medium_tests/test_1000_copies_of_small_files_no_merge.json

```diff
@@ -0,0 +1,23 @@
+{
+  "test_name": "test-1000-copies-of-small-files-no-merge",
+  "table_name": "test_1000_copies_of_small_files_no_merge",
```

Comment:
Let's record the name of the small file in the table_name as well, so instead of `small_files` add something like `valid_4_2`. Also do this for the job name just to be consistent.

Reply:
Good advice. Done.